lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -10803,6 +10803,14 @@ void Emit(ParseNode *pnode, ByteCodeGenerator *byteCodeGenerator, FuncInfo *func
         {
             byteCodeGenerator->EmitSuperCall(funcInfo, pnode, fReturnValue);
         }
+        else if (pnode->sxCall.pnodeTarget->nop == knopImport)
+        {
+            ParseNodePtr args = pnode->sxCall.pnodeArgs;
+            Emit(args, byteCodeGenerator, funcInfo, false);
```

Comment:
>args [](start = 17, length = 4)

Since we're piggybacking off a call node, perhaps we want to assert some restrictions e.g. single parameter